python/pyiceberg/io/pyarrow.py

```diff
@@ -484,21 +488,36 @@ def expression_to_pyarrow(expr: BooleanExpression) -> pc.Expression:
     return boolean_expression_visit(expr, _ConvertToArrowExpression())
 
 
+@lru_cache
+def _get_file_format(file_format: FileFormat, **kwargs: Dict[str, Any]) -> ds.FileFormat:
+    if file_format == FileFormat.PARQUET.value:
+        return ds.ParquetFileFormat(**kwargs)
+    elif file_format == FileFormat.ORC.value:
```

Comment:
We want to remove this, and we can implement ORC in https://github.com/apache/iceberg/pull/7033 because it needs more work.